impeller/docs/faq.md

```diff
@@ -100,3 +100,133 @@
     as development progresses.
 * How do you run `impeller_unittests` with Playgrounds enabled?
   * Specify the `--enable_playground` command-line option.
+* Describe Impeller in a Tweet.
+  * "Impeller is AOT mode for rendering."
+* Why did the Flutter team build Impeller?
+  * The short answer, for consistent performance, which could not be achieved
+    with Skia after years of effort by graphics experts. The longer answer...
+  * Flutter applications, via their use of Skia, were susceptible to the problem
+    of jank (user-perceptible stuttering/choppiness in animations and
+    interactions) due to shader compilation. Such problems were [reported as
```

Comment:
Maybe s/due to shader compilation/due to needing rendering shaders to be compiled on-demand before they could be executed/ or similar?

Reply:
good clarification